aten/src/ATen/Dispatch.h

```diff
@@ -279,3 +283,51 @@ inline void deprecated_AT_DISPATCH_ALL_TYPES_AND_HALF_AND_COMPLEX() {}
         AT_ERROR(#NAME, " not implemented for '", TYPE, "'");                                               \
     }                                                                                                       \
   }()
+
+// ----------------------------------------------------------------------------
+// DEPRECATED MACROS, DON'T USE THESE
+// ----------------------------------------------------------------------------
+
+#define AT_DISPATCH_ALL_TYPES_AND_HALF(TYPE, NAME, ...)                      \
```

Comment:
I dont think these two are being used anywhere. can we just kill them?

Reply:
You're the expert on this :) IIRC, we weren't allowed to remove them because external code might be using them.